addons/website_livechat/static/tests/tours/website_livechat_session_user_changes.js

```diff
@@ -1,54 +1,5 @@
 import { registry } from "@web/core/registry";
 
-registry.category("web_tour.tours").add("website_livechat_login_after_chat_start", {
-    url: "/",
-    steps: () => [
-        {
-            trigger: ".o-livechat-root:shadow .o-livechat-LivechatButton",
```

Comment:
Note: removing this tour as it was a technical limitation, not a real spec. Guest cookie is still present after logout so there is no point in preventing the discussion from continuing once the user logs in.